src/environment.d.ts

```diff
@@ -0,0 +1,11 @@
+declare namespace NodeJS {
+  interface ProcessEnv {
+    PORT: string;
+    DB_PORT: string;
+    HOST: string;
+    DB_USERNAME: string;
+    DB_PASS: string;
+    DB_NAME: string;
+    NODE_ENV: 'development' | 'production';
+  }
+}
```

Comment:
What does this do?

Reply:
This is to safely type `process.env` variables as being `string` rather than `string | undefined`